app/controllers/memberships_controller.rb

```diff
@@ -13,6 +13,26 @@ def create
     end
   end
 
+  def approve
+    @membership = Membership.find_by_id(params[:id])
+    if @membership && @membership.joinable.admin?(current_user)
+      @membership.approve!
+      render 'approve.js'
+    else
+      render 'approve_failure.js'
+    end
+  end
+
+  def deny
+    @membership = Membership.find_by_id(params[:id])
+    if @membership && @membership.joinable.admin?(current_user)
+      @membership.deny!
+      render 'deny.js'
+    else
+      render 'deny_failure.js'
+    end
+  end
+
```

Comment:
How do you feel about breaking these into separate controllers? Like `ApproveMembershipsController#create` and `DenyMembershipsController#create` (or something like that)? In order to stick to only the 7 actions index, show, new, create, edit, update, destroy?